### PR TITLE
Account validation

### PR DESCRIPTION
**Pull request by beingindot**
https://github.com/bisq-network/bisq/issues/3460

Added empty check validation for 
popmoney/uphold/revolut/zelle

bank forms already have required changes with previous PR.

### CODE REVIEW

**Review by chimp1984 — DISMISSED** [review]
utACK

File: desktop/src/main/java/bisq/desktop/components/paymentmethods/ClearXchangeForm.java [hunk 1]
```diff
@@ -87,7 +87,7 @@ public void addFormForAddAccount() {
 
     @Override
     protected void autoFillNameTextField() {
-        setAccountNameWithString(mobileNrInputTextField.getText());
+        setAccountNameWithString(mobileNrInputTextField.getText().trim());
```
Comment (ripcurlx) [inline desktop/src/main/java/bisq/desktop/components/paymentmethods/ClearXchangeForm.java]: That is not necessary anymore as your other PR got merged already.

File: desktop/src/main/java/bisq/desktop/components/paymentmethods/MoneyBeamForm.java [hunk 2]
```diff
@@ -75,7 +75,7 @@ public void addFormForAddAccount() {
 
     @Override
     protected void autoFillNameTextField() {
-        setAccountNameWithString(accountIdInputTextField.getText());
+        setAccountNameWithString(accountIdInputTextField.getText().trim());
```
Comment (ripcurlx) [inline desktop/src/main/java/bisq/desktop/components/paymentmethods/MoneyBeamForm.java]: That is not necessary anymore as your other PR got merged already.

File: desktop/src/main/java/bisq/desktop/components/paymentmethods/PopmoneyForm.java [hunk 3]
```diff
@@ -85,7 +85,7 @@ public void addFormForAddAccount() {
 
     @Override
     protected void autoFillNameTextField() {
-        setAccountNameWithString(accountIdInputTextField.getText());
+        setAccountNameWithString(accountIdInputTextField.getText().trim());
```
Comment (ripcurlx) [inline desktop/src/main/java/bisq/desktop/components/paymentmethods/PopmoneyForm.java]: That is not necessary anymore as your other PR got merged already.

File: desktop/src/main/java/bisq/desktop/components/paymentmethods/RevolutForm.java [hunk 4]
```diff
@@ -102,7 +102,7 @@ private void addCurrenciesGrid(boolean isEditable) {
 
     @Override
     protected void autoFillNameTextField() {
-        setAccountNameWithString(accountIdInputTextField.getText());
+        setAccountNameWithString(accountIdInputTextField.getText().trim());
```
Comment (ripcurlx) [inline desktop/src/main/java/bisq/desktop/components/paymentmethods/RevolutForm.java]: That is not necessary anymore as your other PR got merged already.

File: desktop/src/main/java/bisq/desktop/components/paymentmethods/UpholdForm.java [hunk 5]
```diff
@@ -92,7 +92,7 @@ private void addCurrenciesGrid(boolean isEditable) {
 
     @Override
     protected void autoFillNameTextField() {
-        setAccountNameWithString(accountIdInputTextField.getText());
+        setAccountNameWithString(accountIdInputTextField.getText().trim());
```
Comment (ripcurlx) [inline desktop/src/main/java/bisq/desktop/components/paymentmethods/UpholdForm.java]: That is not necessary anymore as your other PR got merged already.

**Review by ripcurlx — CHANGES_REQUESTED** [review]
NACK - Please see my comments

**Comment by ripcurlx** [comment]
@beingindot Also all your commits need signing and you need to resolve the conflict now that your other PR got merged.

**Comment by beingindot** [comment]
@ripcurlx Resolved conflicts and included changes. pls check.

**Review by ripcurlx — APPROVED** [review]

**Pull request merged**